packages/cubejs-materialize-driver/src/MaterializeDriver.ts

```diff
@@ -59,8 +59,35 @@ export class MaterializeDriver extends PostgresDriver {
        * request before determining it as not valid. Default - 10000 ms.
        */
       testConnectionTimeout?: number,
+
+      /**
+       * Optional cluster name to set for the connection.
+       */
+      cluster?: string,
+
+      /**
+       * SSL is enabled by default. Set to false to disable.
+       */
+      ssl?: boolean | { rejectUnauthorized: boolean },
+
+      /**
+       * Application name to set for the connection.
+       */
+      application_name?: string,
     } = {},
   ) {
+    // Enable SSL by default if not set explicitly to false
+    const sslEnv = process.env.CUBEJS_DB_SSL;
+    if (sslEnv !== undefined && sslEnv !== 'false') {
+      options.ssl = {
+        rejectUnauthorized: sslEnv === 'true'
+      };
+    } else if (options.ssl === undefined) {
+      options.ssl = true;
+    }
```

Comment:
Is this a breaking change to the current behavior?

Reply:
Yes, as of recently, anyone using Materialize in production will need to have `CUBEJS_DB_SSL=true`.